Automattic/woocommerce-payments
# fix: contrast ratio for link within tooltip

Fixes #

#### Changes proposed in this Pull Request









- I noticed the text color of the "Learn more" link is kinda low for the font size ([3.64](https://www.siegemedia.com/contrast-ratio#%23007cba-on-%231e1e1e)); Changing to `#bfdeed` ([11.81](https://www.siegemedia.com/contrast-ratio#%23bfdeed-on-%231e1e1e))
- Since this is an external link, we should probably use the `ExternalLink` component, because it adds the "(opens in a new tab)"  text for screen readers (which was missing before)

Before:
<img width="364" alt="Screenshot 2024-08-16 at 4 30 17 PM" src="https://github.com/user-attachments/assets/71fc82e6-3e9f-4f4b-b981-38a573b9e105">

After:
<img width="327" alt="Screenshot 2024-08-16 at 4 45 01 PM" src="https://github.com/user-attachments/assets/9a1d5b70-550f-4352-8bf2-aec421b1b6cb">


#### Testing instructions





- Go to the settings page at https://wcpay.test/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Click (or hover) the fees next to each payment method
- The color of the link should be updated

-------------------


**Post merge**
